code/__DEFINES/maths.dm

```diff
@@ -0,0 +1,209 @@
+// Credits to Nickr5 for the useful procs I've taken from his library resource.
```

Comment:
it's MATH, not MATHS, spell it correctly!

Reply:
You don't want to start this fight.

Reply:
fuck england, it's math

Reply:
Don't say I didn't warn you.

Reply:
IT'S MATH.  "MATHS" DOESN'T MAKE SENSE

Reply:
It comes from ```Math's``` which is a contraction of ```Mathematics```, over time the apostrophe (symbolising the contraction and loss of ```ematic```) was lost, leaving a new word with the original meaning.

It's just like Don't (```Do Not``` -> ```Do n't``` -> ```Don't```) and Can't (```Can not``` -> ```Can ''t``` -> ```Can't```) and like at the beginning of this sentence, it's (```it is``` -> ```it 's``` -> ```it's```, which is why you write its when ```it``` owns something, not ```it's`` since it's already means the contraction)

So no, you're (there it is again~) quite wrong, ```Maths``` makes plenty of sense, it's just evolved slightly.

You don't contract ```Doctor``` to ```D``` do you?, no, it's ```Dr``` coming from ```D'r```.

Reply:
I'm actually American, I just like the sound of maths better here in this context as a defines file.